HOW_TO_RUN_SAMPLES_WITH_SPRING_BOOT_3.md

```diff
@@ -0,0 +1,28 @@
+# How to run samples with Spring Boot 3?
+
+The current examples are built based on Spring Cloud Azure 4.x by default. If you want to use Spring Cloud Azure 6.x based on Spring Boot 3, this document will tell you how to do it.
+
+## How to make a sample support running with Spring Boot 3?
```

Comment:
The main reader of this file should be customer, not sample maintainer.

So I suggest to change :
```
this document will tell you how to do it.

## How to make a sample support running with Spring Boot 3?
1. It's required to add a document *HOW_TO_RUN_THIS_SAMPLE_WITH_SPRING_BOOT_3.md* to introduce the differences, it should be put together with the sample's *README.md*. The outline and contents of the document are as follows:
```

into:
```
please refer to `HOW_TO_RUN_THIS_SAMPLE_WITH_SPRING_BOOT_3.md` beside each sample's `README.md`. The `HOW_TO_RUN_THIS_SAMPLE_WITH_SPRING_BOOT_3.md` should have contents like this: 
```